src/e2e/tatum.notification.spec.ts

```diff
@@ -19,7 +19,7 @@ import { e2eUtil } from './e2e.util'
 
 // TODO pipeline dont work with API keys
 
-describe.skip('notification', () => {
+describe('notification', () => {
```

Comment:
Is this intentional or are notifications stable in CI?

Reply:
intentional
I didnt changed solana, it is unstable.